pkg/iac-providers/terraform/commons/load-dir.go

```diff
@@ -410,3 +417,45 @@ func (m *ModuleConfig) getChildConfigs() []*ModuleConfig {
 	}
 	return allConfigs
 }
+
+// GetRemoteLocation checks wether the source belongs to the remote module present in downloader cache.
+// cache has key = remoteURL and value = tempDir
+func GetRemoteLocation(cache map[string]string, resourcePath string) (remoteURL, tmpDir string) {
+	dir := filepath.Dir(resourcePath)
+	for k, v := range cache {
+		// this condition will never arise added for safe check
+		if len(v) > 0 {
+			// check dir length is greater than tempDir and file belongs to the same tempDir
+			if len(dir) >= len(v) && v == dir[:len(v)] {
+				return k, v
+			}
+		}
+	}
+	return
```

Comment:
I am wondering, why this return with insufficient return params is not causing a build failure

Reply:
no it won’t because I have used named return params

Reply:
Oh, I see it now :)